### ISSUE TITLE
Failed to Build

### ISSUE THREAD

**Issue by SAOMDVN** [K0lb3/tex2img#4]
Hello, I am having trouble building this module
### Information of my environment:
OS: Windows 7 64bit SP1
Python Version: 3.8.5
PIP Version: 20.3.3
Microsoft Visual C++ Version: 15.0
### Error log: 
```
    running install
    running build
    running build_ext
    building 'tex2img' extension
    creating build
    creating build\temp.win32-3.8
    creating build\temp.win32-3.8\Release
    creating build\temp.win32-3.8\Release\src
    creating build\temp.win32-3.8\Release\src\basisu
    creating build\temp.win32-3.8\Release\src\basisu\transcoder
    creating build\temp.win32-3.8\Release\src\etcpack
    creating build\temp.win32-3.8\Release\src\pvrtc_decoder
    E:\Microsoft Visual Studio\VC\Tools\MSVC\14.28.29333\bin\HostX86\x86\cl.exe /c /nologo /Ox /W3 /GL /DNDEBUG /MD -Id:\python\include -Id:\python\include "-IE:\Microsoft Visual Studio\VC\Tools\MSVC\14.28.29333\include" "-IC:\Program Files (x86)\Windows Kits\NETFXSDK\4.6.1\include\um" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.10240.0\ucrt" /EHsc /Tpsrc\main.cpp /Fobuild\temp.win32-3.8\Release\src\main.obj -std=c++11
    cl : Command line warning D9002 : ignoring unknown option '-std=c++11'
    main.cpp
    C:\Users\User\AppData\Local\Temp\pip-install-e30erjpj\tex2img_836879c5d2fc4641b8a8f99a7ea1cd78\src\basisu\basisu_enc.h(689): error C3861: 'tolower': identifier not found
    src\main.cpp(58): warning C4018: '>=': signed/unsigned mismatch
    src\main.cpp(77): warning C4018: '<': signed/unsigned mismatch
    src\main.cpp(78): warning C4018: '<': signed/unsigned mismatch
    src\main.cpp(114): warning C4018: '>=': signed/unsigned mismatch
    src\main.cpp(119): warning C4018: '>=': signed/unsigned mismatch
    error: command 'E:\\Microsoft Visual Studio\\VC\\Tools\\MSVC\\14.28.29333\\bin\\HostX86\\x86\\cl.exe' failed with exit status 2
```
### Step to reproduce:

1. Open Command Prompt
2. Type `pip install tex2img`

**Comment by K0lb3** [comment]
Fixed it in version 0.9.1.

**Comment by SAOMDVN** [comment]
Thank you very much! Worked beautifully!
